### PR TITLE
Fix violations of and reenable rubocop Style/RedundantBegin.

### PR DESCRIPTION
**Pull request by asherkach**

### CODE REVIEW

**Comment by asherkach** [comment]
@epeach, mind kicking off a CircleCI run? The auto-correct botched it badly enough I want to spend a few minutes sanity checking this PR, no need to review it yet.

**Comment by asherkach** [comment]
Ping @epeach (former DOTD) or @balderdash (current DOTD).

File: large_files.sh [hunk 1]
```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
```
Comment (balderdash) [inline large_files.sh]: I'm guessing this file isn't meant to be here.

Comment (asherkach) [reply large_files.sh]: No idea how it ended up there. Your guess is correct, removed.

**Pull request merged**